mmd_tools/core/pmx/exporter.py

```diff
@@ -931,7 +938,9 @@ def execute(self, filepath, **args):
             self.__exportDisplayItems(root, nameMap)
 
         if self.__copyTextures:
-            tex_dir = os.path.join(os.path.dirname(filepath), 'textures')
+            tex_dir = os.path.dirname(filepath)
+            if not bpyutils.addon_preferences('base_texture_folder', ''):
+                tex_dir = os.path.join(tex_dir, 'textures') 
```

Comment:
Sorry for not explaining properly the logic. The logic is, if the user has not set the preference the default `textures` folder is used. However the code could be refined adding another else in the `__copy_textures` function.

```python
if base_folder != '':
    # ...
else:
    tex_dir = tex_dir_fallback
```